content/roadmaps/114-defi/content/100-basics/102-chain.md

```diff
@@ -1 +1,3 @@
 # Chain
+
+Chain ids or chain itself?
```

Comment:
Chain itself 😅 
Might be worth to mention something about reorgs, 51% attack, etc...